data/datasets/poetry_instruction/prepare.py

```diff
@@ -0,0 +1,151 @@
+import json
+import os
+import random
+
+import pandas as pd
+
+# Read the CSV file into a pandas DataFrame. The CSV file was downloaded from Kaggle, and the source is linked in the README.
+
+CURRENT_DIR = os.path.dirname(__file__)  # Gets directory path of the current python module
+df = pd.read_csv(os.path.join(CURRENT_DIR, "PoetryFoundationData.csv"))
+
+# The data in the CSV file is not formatted correctly, so we need to clean it up.
+df["Title"] = df["Title"].replace("\n", "", regex=True).replace("\r", "", regex=True)
+df["Title"] = df["Title"].str.strip()
```

Comment:
you can also do something like
`df["Title"].apply(lambda x: "".join(x.splitlines()).strip())`